plugins/catalog-backend-module-msgraph/README.md

```diff
@@ -183,3 +183,41 @@ export async function myGroupTransformer(
    }),
  );
 ```
+
+## Dynamically adjust Provider Config
```

Comment:
To avoid duplication I don't think we should include this here. It might be better to have a link here to the Backstage.io docs on this topic. Eventually the extra content here should be moved there but I don't think that should be part of this PR.